code/clue.js

```diff
@@ -1,64 +1,279 @@
-// CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
 const mrGreen = {
   firstName: "Jacob",
   lastName: "Green",
   color: "green",
+  nameCol: "beige",
   description: "He has a lot of connections",
   age: 45,
   image: "assets/green.png",
   occupation: "Entrepreneur"
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
+const profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  nameCol: "beige",
+  description: "With his good looks and charm he has found a dark side being sneaky.",
+  age: 36,
+  image: "assets/plum.png",
+  occupation: "Professor"
+}
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  nameCol: "beige",
+  description: "A movie star who will do anything to stay in the spotlight",
+  age: 25,
+  image: "scarlett.png",
```

Comment:
How come the image for missScarlet isn't in the assets folder like the other ones?